src/components/useraccount/orderdetail/MainDetails.vue

```diff
@@ -0,0 +1,155 @@
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-6">
+        <div class="my-orders-order-dark-box">
+          <div class="my-orders-order-dark-box-title">{{ $t('shippingMethod') }}</div>
+          <div>
+            {{me.order.shippingInfo.shippingMethod.name}}
+            <span v-if="me.order.shippingInfo.shippingMethod.description">
+              - {{me.order.shippingInfo.shippingMethod.description}}
+            </span>
+          </div>
+        </div>
+      </div>
+      <div class="col-sm-6">
+        <div class="my-orders-order-dark-box">
+          <div class="my-orders-order-dark-box-title">{{ $t('paymentDetails') }}</div>
+          <span v-if="me.order.paymentInfo">
+            {{me.order.paymentInfo.payments[0].paymentMethodInfo.method}}
+          </span>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import gql from 'graphql-tag';
+
+export default {
+
+  data: () => ({
+    me: null,
+  }),
+
+  computed: {
+    orderNumber() {
+      return this.$route.params.orderNumber;
+    },
+  },
+
+  apollo: {
+    me: {
+      query: gql`
+        query orderByOrderNumber($orderNumber: String) {
+          me {
+            order(orderNumber: $orderNumber) {
+              shippingAddress {
+                title
+                firstName
+                lastName
+                streetName
+                streetNumber
+                additionalStreetInfo
+                city
+                postalCode
+                region
+                country
+                contactInfo {
+                  phone
+                  email
+                }
+              }
+              billingAddress {
+                title
+                firstName
+                lastName
+                streetName
+                streetNumber
+                additionalStreetInfo
+                city
+                postalCode
+                region
+                country
+                contactInfo {
+                  phone
+                  email
+                }
+              }
+              shippingInfo {
+                shippingMethod {
+                  name
+                  description
+                }
+              }
+              paymentInfo {
+                payments {
+                  paymentMethodInfo {
+                    method
+                  }
+                }
+              }
+            }
+          }
+        }`,
+      variables() {
+        return {
+          orderNumber: this.orderNumber,
+        };
+      },
+    },
+  },
+};
+</script>
+
+<i18n>
+en:
+  shippingAddress: "Shipping address"
+  billingAddress: "Billing address"
+  shippingMethod: "Shipping method"
+  paymentDetails: "Payment details"
+de:
+  shippingAddress: "Lieferanschrift"
+  billingAddress: "Rechnungsadresse"
+  shippingMethod: "Versandart"
+  paymentDetails: "Zahlungsdetails"
+</i18n>
```

Comment:
I would try to split this component into 3: addresses, shipping and payment. It's the same 3 sections we will have in checkout. It may ease up when somebody wants to edit, add or remove one of the sections.